site/content/integrate/admin-guide/admin-slash-commands/_index.md

```diff
@@ -0,0 +1,177 @@
+---
```

Comment:
Can we consolidate this whole page with https://developers.mattermost.com/integrate/slash-commands/?